02_Development/Display/Tests/test_spl06_007.py

```diff
@@ -255,100 +232,124 @@ def test_return_freezing_given_actual_data(self):
                                msg="Fails to return 0degC given data that "
                                "matches that.")
 
-            
-class TestPressureSensor(unittest.TestCase):
+
+class TestCommunicator(unittest.TestCase):
 
     def setUp(self):
-        self._mux_select(0)
-        self._sensor = PressureSensor()
+        # A bug in adafruit_platformdetect causes resource warnings
+        # to come up in unittest, so the warnings must be filtered out
+        # https://github.com/adafruit/Adafruit_Python_PlatformDetect/issues/89
+        warnings.filterwarnings("ignore",
+                                message="unclosed file",
+                                category=ResourceWarning)
+        self._mux_select(4)
+        self._communicator = Communicator()
 
     def tearDown(self):
-        self._sensor.close()
+        self._communicator.close()
```

Comment:
what's the difference between a "sensor" and a "communicator" in this scheme?  I think it would make sense to put some kind of commentary together to explain the difference